README.md

```diff
@@ -1,25 +1,61 @@
-# pe-reports #
+# Posture and Exposure (P&E) Reports #
 
 [![GitHub Build Status](https://github.com/cisagov/pe-reports/workflows/build/badge.svg)](https://github.com/cisagov/pe-reports/actions)
 [![Coverage Status](https://coveralls.io/repos/github/cisagov/pe-reports/badge.svg?branch=develop)](https://coveralls.io/github/cisagov/pe-reports?branch=develop)
 [![Total alerts](https://img.shields.io/lgtm/alerts/g/cisagov/pe-reports.svg?logo=lgtm&logoWidth=18)](https://lgtm.com/projects/g/cisagov/pe-reports/alerts/)
 [![Language grade: Python](https://img.shields.io/lgtm/grade/python/g/cisagov/pe-reports.svg?logo=lgtm&logoWidth=18)](https://lgtm.com/projects/g/cisagov/pe-reports/context:python)
 [![Known Vulnerabilities](https://snyk.io/test/github/cisagov/pe-reports/develop/badge.svg)](https://snyk.io/test/github/cisagov/pe-reports)
 
-This is a generic skeleton project that can be used to quickly get a
-new [cisagov](https://github.com/cisagov) Python library GitHub
-project started.  This skeleton project contains [licensing
-information](LICENSE), as well as
-[pre-commit hooks](https://pre-commit.com) and
-[GitHub Actions](https://github.com/features/actions) configurations
-appropriate for a Python library project.
+This package is used to generate encrypted Posture and Exposure (P&E) pdf
+reports using raw_data.xlsx files.
 
-## New Repositories from a Skeleton ##
+## Device Setup ##
 
-Please see our [Project Setup guide](https://github.com/cisagov/development-guide/tree/develop/project_setup)
-for step-by-step instructions on how to start a new repository from
-a skeleton. This will save you time and effort when configuring a
-new repository!
+Install LibreOffice for powerpoint to pdf conversion
+<https://www.libreoffice.org/get-help/install-howto/macos/>.
+
+Install python 3
+
+(Optional) Setting up your Mac:
+<https://github.com/cisagov/development-guide/blob/develop/dev_envs/mac-env-setup.md>
+
+## Installation ##
+
+Please see the
+[Creating the Python virtual environment](CONTRIBUTING.md#creating-the-python-virtual-environment)
+section of the [CONTRIBUTING](CONTRIBUTING.md) document for information about
+setting up a Python virtual environment.
+
+Required third party libraries: openpyxl, pymongo, pymupdf, pandas, python-pptx
+Install third party libraries using:
+
+```console
+python3 -m pip install LIBRARY
+```
```

Comment:
Is this section still needed, now that these libraries have been included in setup.py and will get automatically pulled in when this project is installed with pip?
https://github.com/cisagov/pe-reports/blob/22a998a1094aaf507dd1a8c2408930b2d25ae897/setup.py#L90-L99

Reply:
Yes, removing this section. At the time it was written these instructions were for the internal PE team prior to any code migration to the skeleton.